flake.nix

```diff
@@ -0,0 +1,31 @@
+# How to setup this project?
+#
+# 1. Install Nix:
+#
+#     curl -sSf -L https://install.lix.systems/lix | sh -s -- install
```

Comment:
You're giving instructions to install a hard fork of nix without telling people that's what you're having them do.

If people are following your instructions here, then they're probably new to nix, so let's do the least surprising thing, and have them install `nix`. I would recommend just linking them to the official (or determinate systems) installation page.